R/transform_pmf_model_out.R

```diff
@@ -0,0 +1,63 @@
+#' Transform pmf model output into a forecast object
+#'
+#' @param model_out_tbl Model output tibble with predictions
+#' @param target_observations Observed 'ground truth' data to be compared against predictions
+#' @param output_type_id_order For nominal variables, this should be `NULL` (the default).
+#' For ordinal variables, this is a vector of levels for pmf forecasts, in
+#' increasing order of the levels.
+#'
+#' @return forecast_quantile
+#' @importFrom rlang .data
+transform_pmf_model_out <- function(model_out_tbl, target_observations, output_type_id_order = NULL) {
```

Comment:
Is this to be a user-facing function? If so, it should have an `@export` directive and examples.

Reply:
no, it will not be user-facing